docs/development/workflow/virtual_pythons.rst

```diff
@@ -164,36 +210,60 @@ addition to activating new ones.
 
       workon ENV
 
-* ` conda`: Activate the environment ``ENV`` with::
+* `conda`_: Activate the environment ``ENV`` with::
 
       conda activate ENV
 
+* `pipenv`_: Activate the environment by ``cd``-ing into the project
+  directory (i.e. the copy of the Astropy repository on your computer) and
```

Comment:
We made the distinction of using capitalized Astropy to indicate the whole project and `astropy` to indicate just this repo.

```suggestion
  directory (i.e., the copy of the ``astropy`` repository on your computer) and
```